script/utils.js

```diff
@@ -1,9 +1,10 @@
 const { spawn } = require('child_process');
 
 const runCommand = cmd => {
-  const child = spawn(cmd, [], { shell: true });
-  child.stdout.pipe(process.stdout);
-  child.stderr.pipe(process.stderr);
+  const child = spawn(cmd, [], { shell: true, stdio: 'inherit' });
```

Comment:
This passes all the output through and maintains the colors, which the old code was not doing. This also affects the watch command, making that output more compact and with some color highlighting